constructor/fcp.py

```diff
@@ -255,17 +255,16 @@ def _main(name, version, download_dir, platform, channel_urls=(), channels_remap
         (x['src'] for x in channels_remap),
     ))
 
-    if environment_file or environment:
```

Comment:
I found the if more likely to catch future developers out than anything else, so I de-dented it - the ``user_conda`` variable is therefore available to all situations (environments or not), even if not used.

If this is not desired, I have no objection to re-indenting it.

Reply:
Perhaps this *does* introduce a behaviour change - you have to have a ``user_conda`` available (in the installed environment) if you want to construct an installer for a non native platform. Previously this limitation was only for ``environment`` based installations.

Reply:
No testing from #342 (at all). Perhaps it is harder than I think, since the test (envs) need to run on multiple platforms...